src/matrix/matrixnormal.jl

```diff
@@ -0,0 +1,161 @@
+"""
+    MatrixNormal{T <: Real, TM <: AbstractMatrix, ST <: AbstractPDMat} <: ContinuousMatrixDistribution
+
+Matrix Normal Distribution
+
+`X ~ MN(M, U, V)`
+
+M: n x p
+
+U: n x n positive definite
+
+V: p x p positive definite
+
+f(X) = c0 * exp( -0.5 tr[inv(V) (X - M)' inv(U) (X - M)] )
+
+c0   = (2pi) ^ {-np / 2} |V| ^ {-n / 2} |U| ^ {-p / 2}
+
+https://en.wikipedia.org/wiki/Matrix_normal_distribution
```

Comment:
This doesn't render right in the REPL - the underscores `_` are treated by Markdown as italics, so we get "Matrix _normal_ distribution" as one word and the link doesn't work. 

Better to put this as a link in the text? i.e. On the first line 
```julia
"""
    MatrixNormal{...}

[Matrix Normal Distribution](https://en.wikipedia.org/wiki/Matrix_normal_distribution)
...
"""

Reply:
Right, I just noticed that. Thanks!